nd4j/nd4j-backends/nd4j-api-parent/nd4j-api/src/main/java/org/nd4j/linalg/api/ops/impl/transforms/pairwise/arithmetic/SquaredDifferenceOp.java

```diff
@@ -40,10 +40,17 @@ public SquaredDifferenceOp(SameDiff sameDiff, SDVariable[] args, boolean inPlace
         super(sameDiff, args, inPlace);
     }
 
+    public SquaredDifferenceOp(SameDiff sameDiff, SDVariable[] args) {
```

Comment:
Let's change this (both everywhere here and in Math.kt) to having 2 inputs: x and y, instead of 1 `count = AtLeast(1)` input.
The op definition is simply `out = (x-y)^2` , i.e., exactly 2 inputs are required

Reply:
Fixed.